### DISCUSSION TITLE
Can't get a logo on mac

### DISCUSSION THREAD

**Discussion by ColtonIdle** [fastfetch-cli/fastfetch#953] [Q&A] [answered]
I'm running in iterm and built-in terminal

`fastfetch --logo "~/Desktop/test.jpeg" --logo-width 1090 --logo-height 1088` but no dice. I even tried the image as a png. I just get `����` for jpg or `�PNG` for png. I feel like im probably noobing this up somehow. am i missing something basic here?

**Comment by CarterLi** [comment, marked as answer]
if you use iterm, you should use iterm protocol. `fastfetch --iterm ~/Desktop/test.jpeg`

**Comment by CarterLi** [comment]
The unit of `--logo-width` and `--logo-height` is number of columns and rows, not pixels.

**Comment by ColtonIdle** [comment]
bah. i am an idiot. thank you!!!

**Comment by CarterLi** [comment]
The built-in Apple Terminal doesn't support image logos.

**Reply by ColtonIdle** [reply]
thanks. that was my next question. now if i can only figure out a way for my mac to not suggest terminal anymore and just suggest iterm... lol
